README.md

```diff
@@ -78,13 +78,24 @@ The above command will automatically fail the build if the average code coverage
 
 ### Excluding From Coverage
 
+#### Attributes  
 You can ignore a method or an entire class from code coverage by creating and applying any of the following attributes:
 
 * ExcludeFromCoverage
 * ExcludeFromCoverageAttribute
 
 Coverlet just uses the type name, so the attributes can be created under any namespace of your choosing.
 
+#### File Path  
+You can also ignore code coverage by specifing files using the `Exclude` property
```

Comment:
Nit: You can also ignore specific source files from code coverage using the `Exclude` property